spec/controllers/user_controller_spec.rb

```diff
@@ -76,7 +76,13 @@ def make_request
 
     end
 
-    context 'when the user being shown is logged in' do
+    context 'when logged in viewing your own requests' do
```

Comment:
Do we need an equivalent 'when logged in viewing another user's requests'? I can only see this and 'not logged in'